Cleanup mock api setup to more easily allow multiple devs (#16)
AmeelioDev/letters-mobile: README.md

### Prerequisites

- Install [Node.js](https://nodejs.org/en/)
- Install [npm](https://www.npmjs.com/get-npm)

> - Install [Expo CLI](https://docs.expo.io/workflow/expo-cli/)

Author: lol i actually don't remember doing this I think my editor might have done it automatically my b

> - Install [Expo CLI](https://docs.expo.io/workflow/expo-cli/)

Author: ohhhhhh it's a linting thing I think. i'll figure out how to change this setting

> - Install [Expo CLI](https://docs.expo.io/workflow/expo-cli/)

Author: okay this is a prettier thing, apparently pretty controversial https://github.com/prettier/prettier/issues/4251

npm install
```

#### Configuring Environment Variables

Create a file .env in the project root. In that file, add the line

> `MOCK_API_IP = [your IPv4 local IP address]`

Reviewer: Does this work if the user just does `MOCK_API_IP = localhost` or do they have to use their actual IP? Just curious.

Author: it does not work unfortunately

> This will make it so that during development your app can successfully connect to the mock API backend.

Reviewer: Probably don't need this line. Open to discussion on this. In general I like to keep the README short and sweet. Not sure this line adds much explanation to new users.

Visit http://localhost:19002/ in your browser and you should see the Metro Bundler page. From here you can follow the instructions in your terminal to launch the app with live reloading on your device. Hooray, you've successfully installed and run the Ameelio Letters mobile application!

> You should also start the mock backend API by running
> 
> ```
> node ./mock-api/app.js
> ```
> 
> in a separate terminal from the root directory.
> 

Reviewer: ```suggestion
Start the mock backend API in another terminal by running

```
node ./mock-api/app.js
```

```

Slightly more concise. This way we keep the format of instruction first then the related command

## Built With

- [React Native](https://reactnative.dev/) - The cross-platform framework used

> - [npm](https://reactnative.dev/) - Dependency Management

Reviewer: Same as above comment, why change these markdown characters?

Author: whoops my b
